src/styles/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  max-width: 200px;
```

Comment:
Per task:

> card width is 200px including border

So  in your case `max-width` will not guarantee card width is always exactly 200px


<img width="1218" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/1983879/ef9b47a0-0562-4381-87d8-f9288a8c8529">

Reply:
Regarding @tiserett comment

You removed height - that is good
But as per task you need card to be exactly 200px, I would say it is ok to have `width` set

Reply:
Regarding @tiserett comment

You removed height - that is good
But as per task you need card to be exactly 200px, I would say it is ok to have width set

Reply:
Regarding @tiserett comment

You removed height - that is good
But as per task you need card to be exactly 200px, I would say it is ok to have width set